#VRML_SIM R2022a utf8
WorldInfo {
  title "WalkingRobot"
}
Viewpoint {
  orientation 0.2855176993486142 0.38398083447596854 -0.8780879011316653 1.4052772827077105
  position -3.6288192270363666 9.499832697428719 7.229717218951675
}
TexturedBackgroundLight {
}
TexturedBackground {
}
Floor {
}
Robot {
  translation 0 0.0899998 0.29
  rotation 1 0 0 1.5708053071795867
  children [
    Hinge2Joint {
      jointParameters HingeJointParameters {
      }
      device [
        RotationalMotor {
          name "LeftLegMotor1"
        }
      ]
      device2 [
        RotationalMotor {
          name "LeftLegMotor2"
          acceleration 3
        }
      ]
      endPoint DEF LeftLeg Solid {
        translation 0.281202 -0.0727914 -6.11053e-07
        rotation 4.100929999998433e-07 -0.9999999999996181 -7.716469999997052e-07 1.5707900000003818
        children [
          DEF LeftLegBody Shape {
            appearance PBRAppearance {
              metalness 0
            }
            geometry Capsule {
              height 0.2
              radius 0.05
            }
          }
        ]
        boundingObject USE LeftLegBody
        physics Physics {
        }
      }
    }
    Hinge2Joint {
      jointParameters HingeJointParameters {
      }
      device [
        RotationalMotor {
          name "RightLegMotor1"
        }
      ]
      device2 [
        RotationalMotor {
          name "RightLegMotor2"
        }
      ]
      endPoint DEF RightLeg Solid {
        translation -0.281202 -0.0727914 -4.34162e-07
        rotation 5.80171999999886e-07 -0.9999999999998035 -2.3779799999995328e-07 1.5707853071797828
        children [
          DEF RightLegBody Shape {
            appearance PBRAppearance {
              metalness 0
            }
            geometry Capsule {
              height 0.2
              radius 0.05
            }
          }
        ]
        name "solid(1)"
        boundingObject USE RightLegBody
        physics Physics {
        }
      }
    }
    Accelerometer {
      name "Accelerometer"
    }
    DEF Body Shape {
      appearance PBRAppearance {
        baseColor 1 0.495445 0.138933
        metalness 0
      }
      geometry Capsule {
        height 0.3
        radius 0.14
      }
    }
  ]
  name "WalkBot"
  boundingObject USE Body
  physics Physics {
  }
  controller "WalkingBot"
}
Robot {
  translation 0 0 5
  name "EvoSupervisor"
  controller "EvolutionSupervisor"
  supervisor TRUE
}
